tests/unit/h/jinja2_extensions/navbar_data_test.py

```diff
@@ -113,7 +114,14 @@ def test_search_url(self, pyramid_request, matched_route, matchdict, search_url)
     @pytest.fixture
     def user(self, factories):
         user = factories.User()
-        user.groups = [factories.Group(creator=user), factories.Group()]
+
+        user.memberships.extend(
+            [
+                GroupMembership(group=group)
+                for group in [factories.Group(creator=user), factories.Group()]
+            ]
+        )
```

Comment:
If I do `user.memberships = [...]` here I run into an SQLAlchemy warning:

```
sqlalchemy.exc.SAWarning: Object of type <GroupMembership> not in session, add operation along 'Group.memberships' will not proceed (This warning originated from the Session 'autoflush' process, which was invoked automatically in response to a user-initiated operation.)
```

I'm not sure why `user.memberships = [...]` and `group.memberships = [...]` work fine elsewhere in this PR, or why `user.memberships.extend([...])` gets rid of the warning :man_shrugging: